README-jakob.md

```diff
@@ -0,0 +1,11 @@
+# Anleitung für die Erweiterungen 
```

Comment:
Please don’t add these explanation in a file you add to the repository. These belong to the PR itself, not the code.

Reply:
Ok I remove the README-jakob.md file.